_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "don't make users think about updating software"
+title: "Don't Make Users Think about Updating Software"
```

Comment:
I typically use Chicago's headline-style capitalization, which capitalizes these terms. But this is a stylistic choice and we don't have a standard on it yet, so your call!